examples/function-pointers/function_pointers.bal

```diff
@@ -26,11 +26,28 @@ function getFunctionPointer() returns (function (int, string) returns float) {
     return test;
 }
 
+function functionWithRestValues(int... r) returns int {
+    int total = 0;
+
+    foreach var item in r {
+        total = total + item;
```

Comment:
```suggestion
        total += item;
```

Reply:
No longer apply, followed this in other places.